gno.land/pkg/gnoland/types.go

```diff
@@ -18,8 +20,86 @@ var (
 	ErrBalanceEmptyAmount  = errors.New("balance amount is empty")
 )
 
+const (
+
+	// unrestricted allows unrestricted transfers.
+	unrestricted BitSet = 1 << iota
+
+	// TODO: validatorAccount marks an account as validator.
+	validatorAccount
+
+	// TODO: realmAccount marks an account as realm.
+	realmAccount
+)
+
+// bitSet represents a set of flags stored in a 16-bit unsigned integer.
+// Each bit in the BitSet corresponds to a specific flag.
+type BitSet uint16
```

Comment:
Lets use uint64 to leave room for plenty more.

Reply:
done